run-tests.pl

```diff
@@ -55,6 +55,8 @@
 
 our $WANT_TLS = 1;  # This is shared with the test scripts
 
+our $BIND_HOST = "localhost";
```

Comment:
I'd group these "shared with test scripts" variables into a block without blank lines, and move the comment to the top of the block, now that there's several.